Dockerfile

```
@@ -12,6 +12,13 @@ FROM python:${PYTHON_VERSION} as run
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV ENGINE=default
+ENV NAME=default
+ENV USER=default
+ENV PASSWORD=default
+ENV HOST=default
+ENV PORT=default
```

These default environment variables in the Dockerfile can cause unexpected behavior. For example, `ENV PORT=default` sets the `PORT` variable to the string `"default"`. In `settings.py`, your code `os.environ.get("PORT", "3306")` will then use this value `"default"` instead of the intended fallback `"3306"`, which will cause a crash if the variable isn't overridden (e.g., in `docker-compose.yml`).

It's better to remove these `ENV` declarations for database credentials and let the application's own defaults in `settings.py` handle cases where the variables are not set during runtime.